src/main/osd/osd.c

```diff
@@ -147,7 +147,7 @@ static bool backgroundLayerSupported = false;
 escSensorData_t *osdEscDataCombined;
 #endif
 
-STATIC_ASSERT(OSD_POS_MAX == OSD_POS(31,31), OSD_POS_MAX_incorrect);
+STATIC_ASSERT(OSD_POS_MAX == OSD_POS(63,31), OSD_POS_MAX_incorrect);
 
 PG_REGISTER_WITH_RESET_FN(osdConfig_t, osdConfig, PG_OSD_CONFIG, 11);
```

Comment:
Should we raise version?

Reply:
Yes. The following recent PR also updated it, so I need to bump it again.

```
commit 22c5bf4eb3f79246a6dbfaaf4eee7654217e1cc1
Author: TonyBlit <tony.cabello@gmail.com>
Date:   Thu Nov 10 04:53:15 2022 +0100

    Added RSNR to the OSD (#11062)
```

Reply:
That's fine. We only need to raise the version once per release (in dev builds I'd assume we are always flashing with "full chip erase"). This way we could prevent bumping these numbers over and over.
A perfectly clean solution would bump this so that you get "force asked" to apply defaults on first connect after flashing a new dev build. But not needed IMO.